documentation/book/assembly-upgrade-resources.adoc

```diff
@@ -8,8 +8,25 @@ For this release of {ProductName}, resources that use the API version `{KafkaApi
 
 The `{KafkaApiVersionPrev}` API version is deprecated in release {ProductVersion}.
 
+This section describes the upgrade steps for the resources.
+
 IMPORTANT: The upgrade of resources _must_ be performed after xref:assembly-upgrade-{context}[upgrading the Cluster Operator], so the Cluster Operator can understand the resources.
 
+.What if the resource upgrade does not take effect?
```

Comment:
I find it a bit strange that it is formulated as a question. I also find it strange that it is on the beginning - it might be that they will do this and miss the other modules which follow.

Reply:
Thanks Jakub. It's like having an FAQ where it's needed. The alternative is to have this info in each procedure or to add as a separate section to show it in the TOC -- which I'm not sure is necessary.